cadence/cadence_types.py

```diff
@@ -1162,9 +1162,16 @@ class DomainInfo:
 class DomainConfiguration:
     workflow_execution_retention_period_in_days: int = None
     workflow_execution_retention_period: int = None
+    # Deprecated
     emit_metric: bool = None
+    # Deprecated
```

Comment:
Deprecated in favor of history_archival_uri

Reply:
Done